test_common/Test_Common_Sorting.hpp

```diff
@@ -0,0 +1,271 @@
+/*
+//@HEADER
```

Comment:
I don't see a perf test, but I see comments on performance. How do you test performance ? If you have a test can you please commit it to perf-test. If you use the unit tests then that is fine.

Reply:
For the numbers I posted, I just use the unit tests with the array sizes temporarily set much higher. It would be good to copy those over into perf tests.